src/commands/export.ts

```diff
@@ -131,7 +131,9 @@ const exportEntities = async (params: ExportEntitiesParams): Promise<void> => {
     } catch (err) {
       logError(
         params,
-        `Failed to export an entity ${chalk.red(def.name)} due to error ${JSON.stringify(err)}. Stopping export...`,
+        `Failed to export an entity ${chalk.red(def.name)} due to error ${
```

Comment:
Put the `message` property here as well to be consistent with the import command.